src/testFileWatcher.ts

```diff
@@ -11,15 +13,47 @@ import { getTestSourcePaths } from './utils/commandUtils';
 
 class TestFileWatcher implements Disposable {
 
+    private testSourcePaths: string[] = [];
     private disposables: Disposable[] = [];
+    private registerListenersDebounce: (() => Promise<void>) & _.Cancelable = _.debounce(this.registerListenersInternal, 2 * 1000 /*ms*/);
 
-    public async registerListeners(): Promise<void> {
+    public async registerListeners(debounce: boolean = true): Promise<void> {
```

Comment:
how about change the default to false? so if you want to register directly, then just call registerListeners(), that looks more natural to me.

Reply:
ok